packages/migration-graph-shared/src/entities/package.ts

```diff
@@ -70,7 +70,17 @@ export class Package implements IPackage {
   constructor(
     pathToPackage: string,
     {
-      excludePatterns = ['dist', 'test', 'tests'],
+      excludePatterns = [
```

Comment:
it would be great if we could optionally configure this list from the CLI OR rehearsal config